### DISCUSSION TITLE
Why the biomass prediction is different while using different "The start date of the simulation" under "clock"

### DISCUSSION THREAD

**Discussion by sunjianxin** [APSIMInitiative/ApsimX#8752] [Q&A]
Dear ApsimX

I have a simulation running for wheat biomass prediction. The sowing date is Sep-27-2021 and we use ApsimX to predict the biomass weight of termination date of May-9-2022. The questions is such prediction changes when using different "The start date of the simulation" under "Clock". Is this supposed to be like that? or what is the recommended "The start date of the simulation" options for our purpose? The attached figure shows such observation that the predicted biomass value on May-9-2022 changes while using different "The start date of the simulation".

![start_of_simulation_2021-2022_plot_0_forAskApsimXQuestion](https://github.com/APSIMInitiative/ApsimX/assets/6218289/675a4f2f-43ad-44db-9cc1-bab795347b21)

Thank you so much!

Jason

**Comment by par456** [comment]
Hi Jason,

Is it possible for you to upload your apsimx file in a zip for us to have a look at? I suspect you are graphing the Biomass value over the start date, as recorded in a report at harvest (May 9), rather than a daily report to see the value of Biomass on each date.

It is fairly common to have a run-up period prior to sowing the crop, which is why the clock start date is different from the first sowing date, and things like the weather during that period will impact the end results. So the amount of time to run prior to sowing is normally dependant on what you are trying to measure.

**Reply by sunjianxin** [reply]
Thank you so much for your reply! That helps me solving the issue!

Best,
Jason
